pkg/v1/providers/ytt/lib/helpers.star

```diff
@@ -98,7 +98,7 @@ def tkg_image_repo_skip_tls_verify():
 end
 
 def tkg_image_repo_ca_cert():
-  return data.values.TKG_CUSTOM_IMAGE_REPOSITORY_CA_CERTIFICATE
+  return data.values.TKG_PROXY_CA_CERT if data.values.TKG_PROXY_CA_CERT else data.values.TKG_CUSTOM_IMAGE_REPOSITORY_CA_CERTIFICATE
```

Comment:
I suggest adding TKG_PROXY_CA_CERT to some clustergen model file (e.g. optional.model) so the effect of the overlay changes can be seen in the generated yaml outputs.

Reply:
Added clustergen tests. https://storage.googleapis.com/tkg-clustergen/918/20211020044536/clustergen.diff.txt